aptos-move/framework/aptos-framework/sources/PropertyMap.move

```diff
@@ -106,6 +106,16 @@ module AptosFramework::PropertyMap {
         };
     }
 
+    public fun generate_string_vector(values: vector<vector<u8>>): vector<String> {
+        let vals: vector<String> = Vector::empty<String>();
+        let i = 0;
+        while ( i < Vector::length(&values)) {
+            Vector::push_back(&mut vals, ASCII::string(*Vector::borrow(&mut values, i )));
```

Comment:
Is converting all values to strings necessary? Both on-chain and off-chain uses of the property map can do their own conversion right?

Reply:
Yes, the conversion here are keys and types in propertyMap, which takes string